File: 프로그래머스/lv1/12912. 두 정수 사이의 합/두 정수 사이의 합.js
function solution(a, b) {
    // 변수 선언
    let answer = 0, max = Math.max(a,b), min = Math.min(a,b);
    // 두 정수 사이의 합 구하기
    for(let i = min; i <= max; i++){
        answer += i;
    }
    return answer;
}

File: Java/프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int Num1 = a;
        int Num2 = b;
        
        if(a > b) {
            Num1 = b; 
            Num2 = a;
        }
        
        for(int i = Num1; i <=Num2; i++) answer += i;
        return answer;
    }
}